skiko/src/commonMain/kotlin/org/jetbrains/skia/Path.kt

```diff
@@ -444,7 +460,8 @@ class Path internal constructor(ptr: NativePointer) : Managed(ptr, _FinalizerHol
     val isFinite: Boolean
         get() = try {
             Stats.onNativeCall()
-            _nIsFinite(_ptr)
+            // TODO For some reason this method returns 0 instead of false in JS target, investigate
+            !!_nIsFinite(_ptr)
```

Comment:
can we do something more intuitive here - the !! de-facto is used more often in context of a different idiom.